src/tools/pub/environment-variables.md

```diff
@@ -22,7 +22,7 @@ Environment variables allow you to customize pub to suit your needs.
   To specify the location of a particular mirror server,
   use the `PUB_HOSTED_URL` environment variable. For example:
 
-```shell
+```terminal
```

Comment:
I don't think we want to mark this one as terminal. We use `terminal` to indicate something executable but it's not in this case.